### ISSUE TITLE
Can't install services on Ubuntu 20 LTS

### ISSUE THREAD

**Issue by lucytallents** [getferdi/ferdi#2000]
**Describe the bug**
I cannot add any services on a fresh install with a new Ferdi account

**To Reproduce**
1. Open Ferdi & log in
2. Click on 'Discover services'
3. Search for a service on the 'Available services' search tab
4. Click on my chosen service - the service button disappears and a revolving icon appears briefly, then the service button reappears and nothing else happens

**Expected behavior**
I expect to be shown a screen to enter my credentials for the chosen service

**Screenshots**
![image](https://user-images.githubusercontent.com/30717212/134922850-9653d2ab-d960-4814-86a5-5137369cb575.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04.3 LTS
 - Browser: Firefox 92.0 64-bit
 - Version: Ferdi version 5.6.2

**Additional context**
I installed Ferdi using snap, but the same behaviour happened last week when I installed using the .deb package.  Likewise, the error persists regardless of whether I sign in to a newly-created Ferdi account, or use my Franz credentials after importing my Franz settings to Ferdi in the browser.

Happy to send log files etc - just tell me what you need and where to find it!  Thanks

**Comment by lucytallents** [comment]
Thank you for responding.  I had previously tried installing the .deb but received an error from the Ubuntu Software manager - I didn't document the error message, I'm afraid, but it's happening with many software since I switched to Ubuntu v20, so not an issue confined to Ferdi.

I since discovered a different way of installing software from .deb files using the `dpkg` command in the terminal, and now have Ferdi working as expected :partying_face:

**Comment by mhatvan** [comment]
Yeah, Ubuntu Software manager is kinda buggy from my experience. 

I recommend installing .deb programs with e.g. `sudo apt install ./ferdi_5.6.3-nightly.16_amd64.deb`, because it handles updating/replacing existing programs, where Ubuntu Software manager has issues.  

Glad that you got it working! :tada:

**Comment by WillyReyno** [comment]
Hi, I'm getting the same behavior on MacOS (Big Sur 11.5.2) : 

![Can't add service](https://user-images.githubusercontent.com/5576482/135717165-222a0bdd-362f-448c-8d0e-00f551d3416f.gif)

**Comment by vraravam** [comment]
@WillyReyno - can you please indicate which version of ferdi you are running? Also, there have been numerous changes since 5.6.2 - I would *strongly suggest* that you try out the latest nightly. (The link can be found in the downloads page)

**Comment by WillyReyno** [comment]
@vraravam I can't install a `.deb` on MacOS 😅  
I have 5.6.2 installed through the `.dmg` file provided in the release tab.

**Comment by vraravam** [comment]
sorry - i based my recommendation of the deb from the title of this ticket.

The other suggestion still stands: Please upgrade to the nightly - I can assure you that its quite stable.

**Comment by WillyReyno** [comment]
Unfortunately I have the same issue with the nightly build...

**Comment by vraravam** [comment]
In that case, please check the permissions of your Ferdi profile folder. (To find out the profile folder location, you can use the buttons in the `Advanced` section in the Settings/Preferences window.
